.github/workflows/CI.yml

```diff
@@ -182,3 +187,84 @@ jobs:
         with:
           key: ${{ steps.build-cache.outputs.cache-primary-key }}
           path: ~/.cache
+      - name: Get Latest Tag
+        id: latest_tag
+        uses: WyriHaximus/github-action-get-previous-tag@v1
+      - name: Check for updated Project version
+        id: version_check
+        shell: julia --color=yes --project {0}
+        env:
+          LATEST_TAG: ${{ steps.latest_tag.outputs.tag }}
+        run: |
+          using Pkg.Types
+          project = read_project("Project.toml")
+          latest_tag = parse(VersionNumber, ENV["LATEST_TAG"])
+          @info "Project version: project.version"
+          @info "Latest tag: latest_tag"
+          make_release = project.version > latest_tag
```

Comment:
this is probably fine for _most_ cases, but there are times when we need to make non-sequential releases.  for instance, if there are multiple major release series that all need the same bugfix released.  i don't think we should worry about that at this point but may need to in the future.

Reply:
very true...in that case we'll want to check that there isn't already a tag for `project.version`